Rk/tfe december fixes (#508)
hashicorp/terraform-docs-common: website/docs/cloud-docs/api-docs/run.mdx

| `data.attributes.refresh`                          | bool                 | true                                               | Specifies whether or not to refresh the state before a plan.                                                                                                                                                                                                                                                                                                                                                                  |
| `data.attributes.refresh-only`                     | bool                 | false                                              | Whether this run should refresh the state without modifying any resources. Mutually exclusive with `is-destroy`.                                                                                                                                                                                                                                                                                                              |

> | `data.attributes.replace-addrs`                    | array\[string]       |                                                    | Specifies an optional list of resource addresses to be passed to the `-replace` flag.                                                                                                                                                                                                                                                                                                                                         |

Author: I'm going to leave this as is, because the way it currently is represented is clear to me and more concise:
![Screenshot 2024-01-08 at 4 43 51 PM](https://github.com/hashicorp/terraform-docs-common/assets/32436232/d2dc4c39-f36a-4dda-8e95-a18f17eaf456)

| `data.attributes.target-addrs`                     | array\[string]       |                                                    | Specifies an optional list of resource addresses to be passed to the `-target` flag.                                                                                                                                                                                                                                                                                                                                          |

> | `data.attributes.variables`                        | array\[{key, value}] | (empty array)                                      | Specifies an optional list of run-specific variable values. Refer to [Run-Specific Variables](/terraform/cloud-docs/workspaces/variables/managing-variables#run-specific-variables) for details.                                                                                                                                                                                                                              |

Author: same as above

> | `data.attributes.plan-only`                        | bool                 | (from configuration version)                       | Specifies if this is a [speculative, plan-only](/terraform/cloud-docs/run/modes-and-options#plan-only-speculative-plan) run that Terraform cannot apply. Often used in conjunction with terraform-version in order to test whether an upgrade would succeed.                                                                                                                                                                  |

Reviewer: ```suggestion
| `data.attributes.plan-only`                        | bool                 |  Depends on configuration version | Specifies if this is a [speculative, plan-only](/terraform/cloud-docs/run/modes-and-options#plan-only-speculative-plan) run that Terraform cannot apply. Often used in conjunction with terraform-version in order to test whether an upgrade would succeed.                                                                                                                                                                  |
```
I'm not sure, but I'm interpreting that "(from configuration version)" means that you can specify the plan type in the configuration and that each version of the configuration can have different types.

Author: I believe "from configuration version" here means that the default value is set by _your_ configuration version, not that the behavior is different depending on the version. So, I'm going to leave as is.

> | `data.attributes.save-plan`                        | bool                 | false                                              | Specifies if this should be a saved plan run. Terraform can apply saved plan runs, but they perform their plan and checks without becoming the workspace's current run (like a plan-only run), and will only become the current run if you confirm them for apply. When creating new [configuration versions](/terraform/enterprise/api-docs/configuration-versions) for saved plan runs, be sure to make them `provisional`. |

Reviewer: ```suggestion
| `data.attributes.save-plan`                        | bool                 | `false`                                              | When set to `true`, the run is executed as a `save plan` run. A `save plan` run plans and checks the configuration without becoming the workspace's current run. These run types only becomes the current run if you confirm that you want to apply them when prompted. When creating new [configuration versions](/terraform/enterprise/api-docs/configuration-versions) for saved plan runs, be sure to make them `provisional`. |
```
I think this is what we mean.

> | `data.attributes.terraform-version`                | string               |                                                    | Specifies the Terraform version to use in this run. Only valid for plan-only runs; must be a valid Terraform version available to the organization.                                                                                                                                                                                                                                                                           |

Reviewer: ```suggestion
| `data.attributes.terraform-version`                | string               | none | Specifies the Terraform version to use in this run. Only valid for plan-only runs; must be a valid Terraform version available to the organization.                                                                                                                                                                                                                                                                           |
```

> | `data.relationships.workspace.data.id`             | string               |                                                    | Specifies the workspace ID where the run will be executed.                                                                                                                                                                                                                                                                                                                                                                    |
> | `data.relationships.configuration-version.data.id` | string               |                                                    | Specifies the configuration version to use for this run. If the `configuration-version` object is omitted, the run will be created using the workspace's latest configuration version.                                                                                                                                                                                                                                        |

Reviewer: ```suggestion
| `data.relationships.workspace.data.id`             | string               | none | Specifies the workspace ID to execute the run in.                                                                                                                                                                                                                                                                                                                                                                    |
| `data.relationships.configuration-version.data.id` | string               | none | Specifies the configuration version to use for this run. If the `configuration-version` object is omitted, Terraform uses the workspace's latest configuration version to create the run .                                                                                                                                                                                                                                        |
```
